packages/runtime-common/helpers/ai.ts

```diff
@@ -328,3 +329,69 @@ export function generateCardPatchCallSpecification(
     };
   }
 }
+
+export function getPatchTool(attachedOpenCard: CardDef, patchSpec: any) {
```

Comment:
hmmm, i thought we were introducing skill cards to hold the prompts for the different commands so that we would not include command specific prompts in the ai bot